docs/admin/authorization.md

```diff
@@ -43,17 +43,25 @@ A request has the following attributes that can be considered for authorization:
   - group (the list of group names the authenticated user is a member of).
```

Comment:
We now pass the entire `user.Info` object, including username, groups, and the `extra map[string][]string` included by the authentication layer.  See the webhook type for detail.